docs/spec/bank/WIP_keeper.md

```diff
@@ -1,3 +1,5 @@
+WORK IN PROGRESS 
```

Comment:
Can you link to the PR so all comments are still around?

Reply:
done